test/test_other.py

```diff
@@ -3296,6 +3296,14 @@ def test_embind_return_value_policy(self):
 
     self.do_runf('embind/test_return_value_policy.cpp')
 
+  @parameterized({
+    '': [[]],
+    'asyncify': [['-sASYNCIFY=1']]
+  })
+  def test_embind_bigint(self, args):
+    self.do_runf('embind/test_bigint.cpp', '1000000000000n\n-1000000000000n',
+                 emcc_args=['-lembind', '-sWASM_BIGINT'] + args)
```

Comment:
Wow I guess we just never had anyone try this before?

Why does this require `-sWASM_BIGINT` to expose the issue?  It looks like that lack of a `long long ` mapping is a problem regardless of this setting?

Reply:
It seems only ASYNCIFY exposed this because it uses the signature to create a DYNCALL with the legacy mode. Without asyncify bigints were working fine.